tests/test_helpers.py

```diff
@@ -32,6 +32,35 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+     ),
+    (["helper", "l_progress", "-v", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+      "    log_url: https://docs.google.com/document/d/1YC_wtW5Q\n"
+      "    team:\n"
+      "        group_members: ascopatz\n"
+      "        collaborators: aeinstein, pdirac\n"
+      "    deliverable:\n"
+      "        due_date: 2021-05-05\n"
+      "        audience: beginning grad in chemistry\n"
+      "        scope: 1. UCs that are supported or some other scope description if it is software\n"
+      "               2. sketch of science story if it is paper\n"
+      "        platform: description of how and where the audience will access the deliverable.  Journal if it is a paper\n"
```

Comment:
nice, I  agree with your choices of scope and platform and audience here.  👍

Reply:
:D